### ISSUE TITLE
Use of self.train_w in TFFMCore.init_loss()

### ISSUE THREAD

**Issue by martincousi** [geffy/tffm#40]
I am trying to understand how `tffm` is working but I can't figure out why `self.loss` is obtained by multiplying `self.loss_function` by `self.train_w` in the class `TFFMCore`. I would have thought that `self.train_w` shouldn't be there...

```
    def init_loss(self):
        with tf.name_scope('loss') as scope:
            self.loss = self.loss_function(self.outputs, self.train_y) * self.train_w
            self.reduced_loss = tf.reduce_mean(self.loss)
            tf.summary.scalar('loss', self.reduced_loss)
```

**Comment by geffy** [comment]
Hi @martincousi,
`self.train_w` is a vector of per-sample importance weights. 
It's essentially the same thing as `sample_weight` param in `sklearn` api

**Comment by martincousi** [comment]
Ah ok! I confused `self.train_w` with `self.w`. Thank you for the quick reply.
